src/exchange/group/delegation/group-delegation.controller.js

```diff
@@ -146,5 +156,40 @@ angular.module('Module.exchange.controllers').controller(
           this.services.navigation.resetAction();
         });
     }
+
+    getDefaultDomain() {
+      const name = _.last(this.currentAccount.split('@'))
+      return {
+        displayName: name,
+        name: name
```

Comment:
You will ended up with an ESLint `object-shorthand` and `comma-dangle` error.
```suggestion
        name,
```

Reply:
fixed